tests/unittest/utils/torch_ref.py

```
@@ -1239,3 +1240,188 @@ def forward_impl(
+            response_max_len: Maximum length for response section (default 1024)
+            pad_token_id: Token ID for padding (default 0)
+        Returns:
+            input_ids: Tensor of shape [batch_size, prompt_max_len + response_max_len]
+            attention_mask: Tensor of shape [batch_size, prompt_max_len + response_max_len]
+            position_ids: Tensor of shape [batch_size, prompt_max_len + response_max_len]
+        """
+        batch_size = len(original_prompts)
+        total_len = prompt_max_len + response_max_len
+
+        for i, (prompt, response) in enumerate(
+                zip(original_prompts, generated_token_ids_list)):
```

Ruff (0.14.10)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)